CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 project(siminusminus)
 cmake_minimum_required(VERSION 2.8)
 
+# Debug macros
+set(CMAKE_CXX_FLAGS_DEBUG -D"_DEBUG")
```

Comment:
Only for curiosity. Why the flag name is NDEBUG and no DEBUG? is more efficient set macros on release than debug?